mpu6050_scripts/poll-temp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+overwrite() { 
+	echo -e "\r\033[1A\033[0K$@"; 
+	}
+
+#Enable MPU6050 chip
+sudo i2cset -y 0 0x68 0x6B 0
+
+while sleep 1; 
+do
+    #Get raw data
+    HB=$(sudo i2cget -y 0 0x68 0x41)
+    LB=$(sudo i2cget -y 0 0x68 0x42)
+
+    #Convert to signed int
+    TEMP=$(( (HB<<8)+LB-65535 ))
+    #Convert to MPU6050 units
+    TEMP=$(( $TEMP*2941+36530000 ))
+    #Get Left side C
+    TEMPHC=$(( $TEMP/1000000 ))
+    #Get Right side C
+    TEMPLC=$(( ($TEMP-($TEMPHC*1000000))/10000 ))
```

Comment:
What do You think about development function with conversation from integer to 6.6 format

Reply:
Please, specify what does it mean 6.6 format.

Reply:
In this case format 6.6 means that we store values as integer and use 0.000001 as unit. So value 123456123456 means 123456.123456 degrees.